composer.json

```diff
@@ -31,7 +31,7 @@
     },
     "autoload-dev": {
         "psr-4": {
-            "bunq\\test\\": "test/"
+            "bunq\\test\\": "tests/"
```

Comment:
Oh, if tests are moving from `test` to `tests`, please also update the test configuration (`phpunit.xml`) :P

Yes, I fixed it in another MR... :)